awx/main/analytics/core.py

```diff
@@ -5,17 +5,17 @@
 import os.path
 import tempfile
 import shutil
-import subprocess
+import requests
 
 from django.conf import settings
-from django.utils.encoding import smart_str
 from django.utils.timezone import now, timedelta
 from rest_framework.exceptions import PermissionDenied
 
 from awx.conf.license import get_license
-from awx.main.models import Job
+from awx.main.models import Job, Credential
 from awx.main.access import access_registry
 from awx.main.models.ha import TowerAnalyticsState
+from awx.main.utils import decrypt_field
```

Comment:
Don't think you need to decrypt the setting value (it should just do it for you automatically).